Opcap fails with changes due to recent merged PRs

## Bug Description
Opcap gives a panic error as the context is not passed to the auditPlan functions


## Steps to Reproduce:
`./bin/opcap check`

## Expected Result
Run as intented.


## Actual Result
```
yoza@yoza-mac opcap % ./bin/opcap check --packages=cloud-native-postgresql
I0908 09:37:49.107143   31637 request.go:601] Waited for 1.010475375s due to client-side throttling, not priority and fairness, request: GET:https://api.yash-opcap.coreostrain.me:6443/apis/storage.k8s.io/v1beta1?timeout=32s
panic: reflect: Call with too few input arguments

goroutine 1 [running]:
reflect.Value.call({0x1064e3360?, 0x14000445600?, 0x13?}, {0x105fc4957, 0x4}, {0x0, 0x0, 0x0?})
        /opt/homebrew/Cellar/go/1.18.4/libexec/src/reflect/value.go:398 +0x1420
reflect.Value.Call({0x1064e3360?, 0x14000445600?, 0x1400013c430?}, {0x0, 0x0, 0x0})
        /opt/homebrew/Cellar/go/1.18.4/libexec/src/reflect/value.go:339 +0x98
github.com/opdev/opcap/internal/capability.(*CapAuditor).RunAudits(0x14000451ce8, {0x106615b50?, 0x1400013c430?})
        /Users/yoza/Desktop/opcap/internal/capability/auditor.go:98 +0x2a0
github.com/opdev/opcap/cmd.glob..func2(0x10721b6e0?, {0x105fc4e17?, 0x1?, 0x1?})
        /Users/yoza/Desktop/opcap/cmd/check.go:66 +0xc8
github.com/spf13/cobra.(*Command).execute(0x10721b6e0, {0x1400042f290, 0x1, 0x1})
        /Users/yoza/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x4ac
github.com/spf13/cobra.(*Command).ExecuteC(0x10721b960)
        /Users/yoza/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x354
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/yoza/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /Users/yoza/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:895
github.com/opdev/opcap/cmd.Execute()
        /Users/yoza/Desktop/opcap/cmd/root.go:29 +0x68
main.main()
        /Users/yoza/Desktop/opcap/main.go:6 +0x20
```

--- a/internal/capability/auditor.go
+++ b/internal/capability/auditor.go
@@ -92,10 +92,12 @@ func (capAuditor *CapAuditor) RunAudits(ctx context.Context) error {
 		// read a particular audit's auditPlan for functions
 		// to be executed against operator
 		for _, function := range audit.auditPlan {
-
 			// run function/method by name
+			// NOTE: The signature for this method MUST be:
+			// func Fn(context.Context) error
 			m := reflect.ValueOf(&audit).MethodByName(function)
-			m.Call(nil)
+			in := []reflect.Value{reflect.ValueOf(ctx)}
+			m.Call(in)
 		}
 	}
 	return nil

